specification/ml/flush_job/MlFlushJobRequest.ts

```diff
@@ -48,7 +48,8 @@ export interface Request extends RequestBase {
      * Specifies to advance to a particular time value. Results are generated
      * and the model is updated for data from the specified time interval.
      */
-    advance_time?: DateString
+    // Also accepts `now` as a value, epoch seconds (< 10 digits) and epoch milliseconds
+    advance_time?: DateTime
```

Comment:
Should we capture the `DateTime | EpochMillis` aspect of this type? The `EpochSeconds` seems like it'd be hacky to annotate.